libs/native-federation-runtime/src/lib/init-federation.ts

```diff
@@ -66,7 +64,7 @@ export async function processRemoteInfo(
   }
 
   const importMap = createRemoteImportMap(remoteInfo, remoteName, baseUrl);
-  addRemote(remoteName, { ...remoteInfo, baseUrl });
+  importShim.addImportMap(importMap);
```

Comment:
#### Remote unnecessary global cache for loading the remotes
The `es-module-shims` is directly used to load modules instead of caching not-updated